strawberry_django_plus/utils/query.py

```diff
@@ -1,21 +1,34 @@
 import functools
-from typing import List, Optional, Set, Type, TypeVar
+from typing import TYPE_CHECKING, List, Optional, Set, Type, TypeVar
 
 from django.contrib.auth import get_user_model
-from django.contrib.contenttypes.models import ContentType
+from django.contrib.auth.models import AbstractUser
 from django.db.models import Exists, F, Model, Q, QuerySet
 from django.db.models.functions import Cast
 from strawberry_django.utils import is_async
 
-from .typing import TypeOrIterable, UserType
+from .typing import TypeOrIterable
+
+if TYPE_CHECKING:
```

Comment:
I think this can be just a:

```python
try:
    from django.contrib.contenttypes.fields import ContentType
except (ImportError, RuntimeError):
    ContentType = None
```

Then in places where using it, you do something like `if ContentType is not None and isinstance(x, ContentType)` instead of just `isinstance(ContentType)`